app/src/main/res/layout/fragment_questionnaire.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        >
+
+    <data>
+
+        <variable
+                name="viewModel"
+                type="io.github.droidkaigi.confsched2017.viewmodel.QuestionnaireViewModel"
+                />
+    </data>
+
+    <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/grey50"
+            >
+
+        <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:padding="@dimen/space_8dp"
+                >
+
+            <android.support.v7.widget.CardView style="@style/BaseCardView">
+
+                <RelativeLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:orientation="vertical"
+                        android:padding="@dimen/space_16dp"
+                        >
+
+                    <TextView
+                            android:id="@+id/questionnaire_how_old"
+                            style="@style/FeedbackTitle"
+                            android:text="@string/questionnaire_how_old"
+                            />
+
+                    <RadioGroup xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: Unexpected namespace prefix "xmlns" found for tag RadioGroup [MissingPrefix]